### DISCUSSION TITLE
Trying to use this code

### DISCUSSION THREAD

**Discussion by Rosen213** [nguyenquyhy/Flight-Tracker-StreamDeck#394] [General]
I am trying to insert this code in to the Flight_Tracker - (L:FSR_S4_PropAutoSwitch, Number) ! (>L:FSR_S4_PropAutoSwitch, Number)  could anyone let me know if this is possible, thanks. ( Sting S4 toggle Prop Auto )

**Comment by nguyenquyhy** [comment]
What are you trying to do here? That exact line won't work in this plugin, but you should be able to read the value `L:FSR_S4_PropAutoSwitch`

**Comment by Rosen213** [comment]
Yes I can read L:FSR_S4_PropAutoSwitch==1 no problem, but I cant Toggle the switch using the Stream Deck, I have Mobiflight setup with a button on a device, and this code works the switch in the cockpit - (L:FSR_S4_PropAutoSwitch, Number) ! (>L:FSR_S4_PropAutoSwitch, Number)  I added it to hubhop but still not working.

**Reply by nguyenquyhy** [reply]
I haven't used MobiFlight for a long time so the following might not be the optimized steps, but it should work:
1. Create new preset in hubhop (or just use whatever there)
2. In MobiFlight > Extras > Huphop > Downloads
3. In MobiFlight > Extras > MSFS2020 > Update events.txt (legacy). I don't really know if this steps is needed but previously we need this file.
4. Press Run in MobiFlight
5. In Stream Deck plugin, you have to put the preset name in Toggle Event in this format `MobiFlight.PMDG_B737_Weather_Radar_Gain_Control_Left_Dec` (add `MobiFlight.` prefix and change all spaces to underscore _)

**Reply by Rosen213** [reply]
Thanks, can confirm this now works. Great news, I can import all my settings for this 3d party aircraft.

**Reply by Rosen213** [reply]
Hi, I have another question, I have a var in the sim using dev mode behaviours, the var (O:DEICE_Engine_1_Position) 1 == and is outputting 0 or 1 if on or off, is this possible in the plugin to use? thanks

**Comment by ThermoLarry** [comment]
Regarding: "MobiFlight.PMDG_B737_Weather_Radar_Gain_Control_Left_Dec" How does that possibly change your needed variable of: "(L:FSR_S4_PropAutoSwitch, Number)" Would you mind please posting the code you got to change this? Thank you very much in advance.
